docs/zh-cn/technical.md

```diff
@@ -10,6 +10,13 @@
 - [black(18.6b4)](https://github.com/ambv/black) ：代码格式化
 - [McCabe](https://pypi.org/project/mccabe/) ：复杂度检测工具
 
+## 开发约定
+
+- 每行不超过 79 个字符。
+- 函数复杂度不超过 14 。
+- 代码覆盖率不低于 90% 。
+- `# flake8: noqa` 应用于整个文件，应当放置在文件开始的部分。
```

Comment:
# flake8 on top will silence the whole file. 

sometimes, it is required to append to a line so as to silence the warning on the line itself. but I recommend that do not use "flake8: noqa" in general. For our __init__.py, it is used but it is an exception.